lib/src/rules/public_member_api_docs.dart

```diff
@@ -59,6 +59,8 @@ documented getters have corresponding undocumented setters. In this case the
 setters inherit the docs from the getters.
 ''';
 
+// TODO(devoncarew): This lint is very slow - we should profile and optimize it.
```

Comment:
End-of-line comments between members will sometimes be moved by the sorter we use. If you care about where this comment is you might want to move it to somewhere where it won't be moved.